pkg/infra/version.go

```diff
@@ -7,17 +7,24 @@ import (
 
 const (
 	programName = "tape"
-	version     = "0.0.2"
+)
+
+var (
+	Version   string = "latest"
```

Comment:
version, built time, I am not sure for commitSHA.
I don't like update built time for each release. and make it had code here.